packages/tailwindcss/src/candidate.ts

```diff
@@ -189,7 +189,7 @@ export type Candidate =
    * E.g.:
    *
    * - `underline`
-   * - `flex`
+   * - `box-border`
```

Comment:
Not that it matters, but what was the idea behind this change? Just an example with a special character like `-`?

Reply:
Haha `flex` is not a static utility but a functional one now. I put it as a test case and got confused for a second why it didn't work while I implemented my initial version of the printer and then noticed oh it's cause it's functional. I thought I update it just so it's more correct 🙈

Reply:
Hah right, forgot about that. Good catch!